Sources/App/Core/RecentRelease.swift

```diff
@@ -65,7 +66,7 @@ extension RecentRelease {
     }
     
     static func fetch(on database: Database,
-                      limit: Int = Constants.recentPackagesLimit,
+                      limit: Int = Constants.recentReleasesLimit,
```

Comment:
Interesting bug discovered by periphery: we were using the wrong constant (although the value is 7 for both of them).